torat_client/netclient.go

```diff
@@ -34,7 +34,7 @@ func connect(dialer *tor.Dialer) (net.Conn, error) {
 
 // NetClient start tor and invoke connect
 func NetClient() {
-	log.Println("NetClient")
+	log.Println("[NetClient] Starting Tor connection...")
```

Comment:
Tor is started later in line 52

Reply:
Would `[NetClient] Starting client connection...` be better?

Reply:
You could also move it just before the tor.Start

Reply:
Okay will do that